travis/merge_debs_ghpages.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+ls -lh
+# Travis only clones the master branch, so I have to clone everything again to make sure I can switch branches.
+mkdir merging_dir
+git clone https://github.com/randomshinichi/qrllib.git merging_dir/qrllib
```

Comment:
Should this be a configuration setting / environment variable? At the moment is uploading to randomshinichi